### DISCUSSION TITLE
Harvester: plot has a farmer public key that is not in the farmer's pk list

### DISCUSSION THREAD

**Discussion by jjbaker4** [Chia-Network/chia-blockchain#6811] [Support]
This is a callback to the closed issue 2255. After reading the thread there, I was unable to find a solution.

When running Chia plots check on my harvester/plotter (not my main node), I get
"Plot /media/jesse/dest/.plot has a pool public key that is not in the farmer's pool pk list" for any plot that was created on this machine. This was not a problem when I was using Windows on this machine, but it is happening since I switched to Ubuntu. Just for clarity, these plots are most definitely not farming. Interestingly, the plots I created on this machine in Windows are actually still farming.
When I set it up on Windows, and when I set it up on Ubuntu, I followed the directions here:
https://github.com/Chia-Network/chia-blockchain/wiki/Farming-on-many-machines

Thanks for any help you can provide.

**Comment by loppefaaret** [comment]
if you check the `debug.log` file under `~/.chia/mainnet/log/` - do you have any WARNING messages from the harvester, right after launching it ? such as not adding plots

**Reply by jjbaker4** [reply]
Nope, nothing like that there. 
But, there is good news. I shut it all down, turned off upnp, restarted everything, and now it's working.  I'm closing the issue. Thanks.

**Reply by jjbaker4** [reply]
Wait, how do I close the issue? I've done it before, but I don't see anything here. Is this just a discussion, and not an issue? Lol! I must be losing my mind!

**Reply by loppefaaret** [reply]
there should be a little checkmark button hiding, usually for me in the top righ corner, that marks it as answered :)

**Reply by jjbaker4** [reply]
Thanks.

**Comment by BrandtH22** [comment]
@jjbaker4 , based on your previous message it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
